docs/index.asciidoc

```diff
@@ -23,6 +23,46 @@ include::{include_path}/plugin_header.asciidoc[]
 
 Ingest events from the Twitter Streaming API.
 
+Example:
+[source,ruby]
+    input {
+      twitter {
+        consumer_key => '...'
+        consumer_secret => '...'
+        oauth_token => '...'
+        oauth_token_secret => '...'
+        keywords => [ 'logstash' ]
+      }
+    }
+
+Sample event fields generated:
+
+[source,ruby]
+-----
+    {
+        "@timestamp" => 2019-09-23T16:41:53.000Z,
+        "message" => "I forgot how fun it is to write @logstash configs !!! Thank you @jordansissel and @elastic !!!"
+        "user" => "missnebun",
+        "in-reply-to" => nil,
+        "retweeted" => false,
+        "source" => "http://twitter.com/missnebun/status/1176174859833004037",
+        "user_mentions" => [
+            { "screen_name"=>"logstash", "name"=>"logstash", "id"=>217155915 },
+            { "screen_name"=>"jordansissel", "name"=>"@jordansissel", "id"=>15782607 },
+            { "screen_name"=>"elastic", "name"=>"Elastic", "id"=>84512601 }],
+        "symbols" => [],
+        "hashtags" => [],
+        "client" => "<a href=\"http://twitter.com/download/iphone\" rel=\"nofollow\">Twitter for iPhone</a>"
+    }
+-----
+
+[id="plugins-{type}s-{plugin}-ecs"]
+==== Compatibility with the Elastic Common Schema (ECS)
+
+Twitter streams are very specific and do not map easily to the {ecs-ref}[Elastic Common Schema (ECS)].
+We recommend setting a <<plugins-{type}s-{plugin}-target>> when <<plugins-{type}s-{plugin}-ecs_compatibility,ECS compatibility mode>> is enabled.
+The plugin issues a warning in the log when a `target` isn't set.
+
```

Comment:
Nice description here. Provides helpful context explaining why that mapping is tricky. Thank you.